system/core/pages.php

```diff
@@ -534,3 +567,60 @@ function image_resize($src, $dst, $width, $height) {
 		}
 		return true;
 	}
+
+	function getButtonInfo($_args) {
+		/* 0 = buttonName
+		*  1 = buttonName
+		*  2 = language (or null)
+		*/
+		Global $O_O;
+		$_button = "";
```

Comment:
Why use _ as a prefix here? These are often used to mark values as "do not use externally", but since these are local variables, they are already confined to this function only?